pandas/tools/tests/test_util.py

```diff
@@ -291,6 +291,79 @@ def test_non_hashable(self):
         with self.assertRaisesRegexp(TypeError, "Invalid object type"):
             pd.to_numeric(s)
 
+    def test_downcast(self):
+        mixed_data = ['1', 2, 3]
+        int_data = [1, 2, 3]
+        date_data = np.array(['1970-01-02', '1970-01-03',
+                              '1970-01-04'], dtype='datetime64[D]')
+
+        invalid_downcast = 'unsigned-integer'
+        msg = 'invalid downcasting method provided'
+
```

Comment:
it doesn't matter because we get notified it some does fail - generally the differences are pretty small

and we have a way more comprehensive test suite that numpy - it takes more time because it tests a heck of a lot more things and configs